keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/NotificationEventListener.kt

```diff
@@ -279,12 +298,57 @@ class NotificationEventListener(
             deliveryArtifact = deliveryArtifact,
             stateUid = currentState.uid
           ),
-          Type.MANUAL_JUDGMENT,
+          MANUAL_JUDGMENT_AWAIT,
           environment.name)
       }
     }
   }
 
+  @EventListener(VerificationCompleted::class)
```

Comment:
@robfletcher @lorin I used the existing `VerificationCompleted` event here -- if the status is either passed/failed, we will send a notification accordingly. Let me know if I missed something here.